Dockerfile

```diff
@@ -14,6 +14,6 @@ RUN cabal v2-build --only-dependencies -j4
 
 # Add and Install Application Code
 COPY . /opt/ldap-scim-bridge
-RUN cabal v2-install 
+RUN cabal v2-install
```

Comment:
Are you sure the the `v2-*` prefix is still needed? The new cabal ui should be standard now, shouldn't it?

Reply:
yes, but it's not hurting either.